.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: publish
```

Comment:
this file is almost the same as tartufo one (another godaddy opensource python package) [here](https://github.com/godaddy/tartufo/blob/main/.github/workflows/publish.yml), without publishing to docker hub.